src/main/java/com/ogjg/daitgym/UsersChattingRoom.java

```diff
@@ -0,0 +1,47 @@
+package com.ogjg.daitgym;
+
+import jakarta.persistence.*;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+import static lombok.AccessLevel.PROTECTED;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = PROTECTED)
+public class UsersChattingRoom {
+
+    @EmbeddedId
+    private UsersChattingRoomPk id;
+
+//    TODO User 테이블 매핑하기
+//    @MapsId("memberId")
+//    @ManyToOne(fetch = FetchType.LAZY)
+//    @JoinColumn(name = "MEMBER_ID")
+
+    @MapsId("roomId")
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "chat_room_id")
+    private ChatRoom chatRoom;
```

Comment:
```suggestion
//    TODO User 테이블 매핑하기
    @MapsId("memberId")
    @ManyToOne(fetch = LAZY)
    @JoinColumn(name = "MEMBER_ID")
    private User user;

    @MapsId("roomId")
    @ManyToOne(fetch = LAZY)
    @JoinColumn(name = "chat_room_id")
    private ChatRoom chatRoom;
```
이런 식으로 하면 연결이 될 것 같습니다!